ekncontent/ekncontent/eknc-domain.c

```diff
@@ -858,13 +891,27 @@ eknc_domain_query (EkncDomain *self,
 
   GTask *task = g_task_new (self, cancellable, callback, user_data);
   QueryState *state = g_slice_new0 (QueryState);
+  state->models = NULL;
```

Comment:
Should not be necessary since the previous line already zeroes out the whole struct